webapp/commonComponents/profilePicture.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import User from '../../common/user/user'
+
+import { useProfilePicture } from './hooks'
+
+import * as AppState from '../app/appState'
+
+const ProfilePicture = ({ user }) => {
```

Comment:
I would pass the userUuid from the component caller. This component can only render the logged in user profile picture, and that's not what we want; we might want to render also other users pictures

Reply:
Passing userUuid